test/unit/index.js

```diff
@@ -183,10 +194,35 @@ describe('minimal-cipher', function() {
         result.should.deep.equal(obj);
       });
 
-      it('should decrypt a stream', async function() {
-        const data = getRandomUint8();
-        const chunks = await encryptStream({data});
-        chunks.length.should.be.gte(0);
+      it('should decrypt a stream with chunkSize 1 byte', async function() {
+        const data = getRandomUint8({size: 100});
+        const chunks = await encryptStream({data, chunkSize: 1});
+        chunks.length.should.eql(100);
+        for(const chunk of chunks) {
+          chunk.jwe.should.be.a.JWE;
+        }
+        const result = await decryptStream({chunks});
+        result.length.should.be.gte(0);
+        result.should.deep.eql(data);
```

Comment:
`eql` is a shorthand for `deep.equal` already.

Reply:
I actually continue to feel that `deep.equal` is better than `eql` which requires devs to google it, wasting time, each timr they run into it. I did make the change, but I do think we should allow `deep.equal` in test suites. https://github.com/digitalbazaar/minimal-cipher/pull/29/commits/5583046fb336464f46cb5ce7b5dd14de660d52ad